agent/csi/mod_csi_docker.go

```diff
@@ -386,13 +579,30 @@ func (d *dockerBridge) List() (*dvol.ListResponse, error) {
 			continue
 		}
 
+		if _, ok := volMap[name]; ok {
+			d.ctx.WithField("volume", vi).Warn(
+				"docker-csi-bridge: List: skipped volume already in cache")
+			continue
+		}
+
+		d.ctx.WithField("volume", vi.Id.Values).WithField(
+			"name", name).Debug(
+			"docker-csi-bridge: List: found new volume")
```

Comment:
I think the code from here up to the check if the name is in the map can be completely removed now.